crypto/cmp/cmp_msg.c

```diff
@@ -334,10 +334,10 @@ OSSL_CMP_MSG *ossl_cmp_certReq_new(OSSL_CMP_CTX *ctx, int type, int err_code)
             goto err;
         }
         if ((crm = crm_new(ctx, type, OSSL_CMP_CERTREQID, rkey)) == NULL
-                || !OSSL_CRMF_MSG_create_popo(crm, privkey, ctx->digest,
-                                              ctx->popoMethod)
-                /* value.ir is same for cr and kur */
-                || !sk_OSSL_CRMF_MSG_push(msg->body->value.ir, crm))
+            || !OSSL_CRMF_MSG_create_popo(crm, privkey, ctx->digest,
+                                          ctx->popoMethod)
+            /* value.ir is same for cr and kur */
+            || !sk_OSSL_CRMF_MSG_push(msg->body->value.ir, crm))
```

Comment:
why this change?

Reply:
As @Akretsch wrote above when introducing this PR, it includes also some improvements in code formatting (triggered by #10363), logging and internal documentation.
There change you are referring to here is part of the commit with message:
```
    fix various formatting nits in CMP contribution chunks 1-6 found by the new util/check-format.pl
    correct wording in doc, comments, and parameter names: self-signed -> self-issued where appropriate
```
and apparently was meant to improve code formatting and silence a finding by the new `check-format.pl` tool.

Reply:
Revert this.  Since this is a logical expression it is okay to indent the lines beginning with ||

Please see the e-mail discussion on the project mailing list:
https://www.mail-archive.com/openssl-project@openssl.org/msg00164.html

Maybe update your script to at least accept both variants, while new code
should follow the style guide more strictly, I see no need to change everything at once.

Reply:
The mail archive link you provided works only partially for me, but I was already aware of the convention that `&&` and `||` may (but is not required to) have an extra indent. 
Although (for whatever reason) this new convention is still not part of the official coding style https://www.openssl.org/policies/codingstyle.html the tool https://github.com/openssl/openssl/pull/10363 already supports it.

Yet there is a specific problem here: 
between the two lines starting with `||` there is a one-line comment, where it is not clear if the exceptional indentation of the line before/after should be carried over to the comment line. The tool (so far) does not support this potential extra indentation for the comment, and I do not plan to add such a special feature unless the coding guidelines explicitly demand it. 

When @Akretsch prepared the above mentioned commit (which I amended later, that's why you see my name) he apparently took the simple way out and removed the extra indentation for all three lines.
Although his change is fully in line with https://www.openssl.org/policies/codingstyle.html 
I've just reverted this change as you requested.

Reply:
Regarding the one-line comment, if it is referring to the expression below,
please indent it too.

Reply:
We'll try to keep this in mind.

Reply:
Ok, thanks.  Regarding the coding style document, it looks like
it offers way too much freedom, at least when I do a review I need
to follow a specific style at least for the new code added.  However
I would not use mix the PR with new code and style-fixes on existing
code.  Since it is already some 67 files right now.

Reply:
Sorry for the large number of files this time in the PR.
The actual chunk 7 (CMP verify) would just a few files, but over the last two months various improvements on the earlier chunks have come up, partly related to the pre-review on https://github.com/mpeylo/cmpossl/pull/199.
So we thought it would make sense to do these as separate commits but as a prefix to the commit actually adding the new chunk 7 material, trying to keep up with the timeline https://github.com/mpeylo/cmpossl/wiki#planning-and-current-status-of-the-contribution-chunks where the status is already behind schedule again.

Reply:
Regarding the outdated coding style document I've just filed an issue: #10725.

Reply:
This specific issue has been resolved /by returning to the original formatting).